src/extensions/replay/sessionrecording.ts

```diff
@@ -36,6 +44,7 @@ import { gzipSync, strFromU8, strToU8 } from 'fflate'
 
 const BASE_ENDPOINT = '/s/'
 
+const ONE_MINUTE = 1000 * 60
 const FIVE_MINUTES = 1000 * 60 * 5
```

Comment:
very picky... and will have almost no impact individually but a little code golf helps the bundle size over time so you could have 

```suggestion
const FIVE_MINUTES = ONE_MINUTE * 5
```

i tend to dislike that style of writing and would rather fix the bigger problems in the bundle sizing but little things like that tend to help (although the bundler can behave surprisingly 🤣)